File: forca/emJavascript/forcaComUsoDeFuncao.js
// Função para calcular a força
function calcularForca(massa, aceleracao) {
    return massa * aceleracao;
}

// Chamando a função para calcular a força
const forca = calcularForca(massa, aceleracao);

// Variáveis
let massa = 10; // Massa em kg
let aceleracao = 5; // Aceleração em m/s^2

// Exibindo o resultado
console.log("Força = " + forca + "N");

/* 
 
// Função para calcular a força
function calcularForca(massa, aceleracao) {
    return massa * aceleracao; }    

// Exibindo o resultado
console.log("Força = " + calcularForca(5, 10) + "N");

 */

File: forca/emJava/ForcaComUsoDeFuncao.java
package forca.emJava;

public class ForcaComUsoDeFuncao {

    public static double calcularForca(double massa, double aceleracao) {
        return massa * aceleracao;
    }

    public static void main(String[] args) {
        System.out.println("Forca = " + calcularForca(10, 5) + "N");
    }
}
